.github/workflows/bats.yaml

```diff
@@ -0,0 +1,17 @@
+name: Bats tests
+on: [push, pull_request]
+env:
+  functional_test: 'true'
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
```

Comment:
Alternatively `ubuntu-latest` will keep at the latest image; I personally prefer explicit versions, to have more control over when exactly to migrate. Your project, your choice.